src/tools/sssctl/sssctl_config.c

```diff
@@ -64,6 +64,9 @@ errno_t sssctl_config_check(struct sss_cmdline *cmdline,
     /* Open config file */
     ret = sss_ini_config_file_open(init_data, SSSD_CONFIG_FILE);
     if (ret != EOK) {
+        if (ret == ENOENT) {
+            ERROR("SSSD configuration file does not exist.\n");
+        }
```

Comment:
Yes, it calls gettext, the definition is in src/util/debug.h:
#define ERROR(fmt, ...) fprintf(stderr, gettext(fmt), ##__VA_ARGS__)